src/components/pages/contact/leaflet/leaflet.tsx

```diff
@@ -60,8 +61,17 @@ const MapWrapper = styled.div`
 export const Leaflet = () => {
   const [mapInstance, setMapInstance] = useState(null);
   const [isBrowser, setIsBrowser] = useState(false);
+  const [zoomLevel, setZoomLevel] = useState(MAP_VIEW_ZOOM_DESKTOP);
+
   useEffect(() => {
     setIsBrowser(true);
+
+    const { innerWidth: width } = window;
+    if (width < 960) {
+      setZoomLevel(MAP_VIEW_ZOOM_MOBILE);
+    } else {
+      setZoomLevel(MAP_VIEW_ZOOM_DESKTOP);
+    }
```

Comment:
1. using window will break SSR. @feedm3 correct?
2. This will not update when the browser window changes. Which is actually not a big problem. But what about using `useMedia` which we already use to fetch the ratio. You can use it do determine the size like a breakpoint with max-width etc.

Reference: 
https://github.com/satellytes/satellytes.com/blob/eae2810c723badda85ad797899d27853cb36c02a/src/components/content/heroes/blog-hero.tsx#L56

Reply:
I shared the Preview URL with Eric and he said that it is should have the same zoom level on mobile and desktop now. I will remove the logic. Thanks anyway for your feedback @georgiee 🙏

Reply:
You can use the window object within `useEffect`. It only gets executed on the client.

`useMedia` would be the better solution in general.